### ISSUE TITLE
Default order of indirectly-linked sentences

### ISSUE THREAD

**Issue by soliloquist-tatoeba** [Tatoeba/tatoeba2#1907]
When a sentence has more than several indirectly-linked translations, the rest are hidden. I think it would be better if the indirectly-linked sentences in the same language were at the top. Many people would more likely be interested in alternative translations of the sentence they view than sentences in other languages. A touch of rearrangement (exception) might be useful here.

<img width="835" alt="inlink" src="https://user-images.githubusercontent.com/47579687/59806565-77744300-92fd-11e9-8296-92dac6fcce5b.png">

**Comment by ckjpn** [comment]
At first glance, this seems like a good idea to me, but perhaps should be limited to logged-in members, since this would likely be very confusing for first-time visitors.

Maybe this could be an opt-in feature like some of our other [settings](https://tatoeba.org/eng/user/settings), allowing members to decide for themselves whether they want this.

**Comment by soliloquist-tatoeba** [comment]
I'm not sure if it would be confusing for first-time visitors, but as long as it's provided to users, it doesn't matter for me whether it would be the default behavior or an opt-in feature.

**Comment by soliloquist-tatoeba** [comment]
I noticed that a setting we already have can be used to hide sentences in languages other than the ones we are interested in.

![Settings](https://user-images.githubusercontent.com/47579687/59968508-906e3580-9543-11e9-9245-1f7eab73e8ff.png)

That's a different feature and using it would sacrifice some functionality, but it can be used as a temporary solution.

**Comment by jiru** [comment]
I think the real problem here is that there are too much translations displayed, so it is hard to find out the ones that are relevant the you. One possible way to solve this problem is to [hide translations in languages that are not in your profile](https://tatoeba.org/fra/wall/show_message/32098#message_32098).

**Comment by soliloquist-tatoeba** [comment]
> One possible way to solve this problem is to [hide translations in languages that are not in your profile](https://tatoeba.org/fra/wall/show_message/32098#message_32098).

@jiru  
Yes, that would be great. You can close this issue if you're planning to implement such a feature.

**Comment by trang** [comment]
I think we can keep this issue open because the order of the translations does matter. There's some work to do on this topic. 

But we cannot assume that the user will want to see in priority indirect translations that are in the same language as the main sentence. They could not care at all about same-language translations.

Hiding translations in languages that are not in your profile is a solution for another issue. It does help in this current issue, but it's more about solving the problem that one day some sentences will have thousands of translations and it will be more and more tiring to spot the translations you're interested in.
